src/Behat/Config/Formatter/JUnitFormatter.php

```diff
@@ -4,12 +4,19 @@
 
 namespace Behat\Config\Formatter;
 
+use PhpParser\Node\Expr;
+
 final class JUnitFormatter extends Formatter
 {
     public const NAME = 'junit';
 
-    public function __construct()
+    public function __construct(...$baseOptions)
```

Comment:
Previously these named formatters would not accept any settings as array values. This would not allow us to pass any setting other than those defined in the constructors unless we used functions. In all other cases using functions is optional, so I added the capability for these formatters to accept further options. This is also needed to be able to build them when we are building them to generate the PHP expr